api/src/functions/express/app.ts

```diff
@@ -235,9 +235,10 @@ const FORMATION_API_BASE_URL =
     : process.env.FORMATION_API_BASE_URL || `http://${IS_DOCKER ? "wiremock" : "localhost"}:9000`;
 
 const GOV2GO_REGISTRATION_API_KEY = process.env.GOV2GO_REGISTRATION_API_KEY || "";
-const GOV2GO_REGISTRATION_BASE_URL = IS_OFFLINE
-  ? `http://${IS_DOCKER ? "wiremock" : "localhost"}:9000`
-  : process.env.GOV2GO_REGISTRATION_BASE_URL || "";
+const GOV2GO_REGISTRATION_BASE_URL =
+  process.env.USE_WIREMOCK_FOR_GET_TAX_CALENDAR_SEARCH?.toLowerCase() === "true"
```

Comment:
[pebble] I see what you're trying to do here, but this isn't a pattern we follow anywhere else. For consistency sake, let's remove this.
```suggestion
  process.env.USE_WIREMOCK_FOR_GET_TAX_CALENDAR_SEARCH === "true"
```

Reply:
Thanks for the feedback @jphechter . I understand your concern about mantaining consistency across the codebase and I agree that it is important. That said, this pattern in question is already being used elsewhere in the file for both `FORMATION_API_BASE_URL` and `BUSINESS_NAME_BASE_URL`. Happy to refactor these as well if thats what we want to go for.

Reply:
changed